README.md

```diff
@@ -1,55 +1,22 @@
 # schwifty-migration
 
-#### An [Objection.js](https://github.com/Vincit/objection.js) model migration tool for use during development
+#### An [Objection.js](https://vincit.github.io/objection.js) model migration tool for use during development
```

Comment:
Small peas, but we have been linking to Objection's github elsewhere in the pal-verse.